### PR TITLE
Arguments handling for classDb functions 2/2

### LINKED ISSUE #1681: Multiple arguments callback function in diagrams
I don't don't understand why only the Gantt diagram handles callback function with parameters. I'm more interested in seeing this feature on the flowchart diagram but there is there a particular reason to to allow it for the others. 
It would be nice to have a unique way to handle click events in diagram. 
Would it be possible to have a common way to do that? 

For example, it would be possible to write this on every diagram 
```
click cl2 call printArguments("test1", "test2", test3) 
click cl3 call printTask()
```

### PR DESCRIPTION
**Pull request by mmorel-35**
## :bookmark_tabs: Summary
This PR allows to use parameter with callback functions in class diagrams

Resolves #1681 

## :straight_ruler: Design Decisions
I have aligned the way href and functions are called to the Gantt Diagram's way to allow to add parameters to functions in the class diagrams. It also harmonize the way callback functions and links are defined

### :clipboard: Tasks
Make sure you
- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md) 
- [x] :computer: have added unit/e2e tests (if appropriate) 
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

File: src/diagrams/class/parser/classDiagram.jison [hunk 1]
```diff
@@ -29,8 +37,7 @@
 
 "class"               return 'CLASS';
 "cssClass"            return 'CSSCLASS';
-"callback"            return 'CALLBACK';
-"link"                return 'LINK';
+"click"               return 'CLICK';
```
Comment (jgreywolf) [inline src/diagrams/class/parser/classDiagram.jison:40]: Thank you for this part.  I was originally having issues with getting this to work correctly, which is why I separated them - and always intended to go back and fix it but never got around to it.

**Comment by mmorel-35** [comment]
Shall we add the capability to change the target of the link as it is done on the flowchart diagram in this PR ?

**Comment by knsv** [comment]
@mmorel-35 Thanks again for this. The comment in your other PR applies here as well.

File: src/diagrams/class/svgDraw.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ export const drawClass = function(elem, classDef, conf) {
     title = g
       .append('svg:a')
       .attr('xlink:href', classDef.link)
```
Comment (mmorel-35) [inline src/diagrams/class/svgDraw.js:169]: Anyone knows why it is `xlink:href` and not just `href` ?

**Pull request merged**